File: Student/src/main/java/com/quick/Student.java
package com.quick;

public class Student {
    String name;
    int english;
    int math;

    public Student(String name, int english, int math) {
        this.name = name;
        this.english = english;
        this.math = math;
    }

    public int Highest() {
        return (math > english) ? math : english;
    }

    public void print() {
        System.out.println(name + "\t" + english + "\t" + math);
    }

}

File: Student/src/main/kotlin/com/quick/Student.kt
package com.quick

class Student(var name:String, var english:Int, var math:Int) {
    fun average() : Int {
        return (english+math)/2
    }

    fun highest() = if (english > math) english else math
}

fun main(args: Array<String>) {
    val stu = Student("Quick", 80, 100)
    println(stu.highest())
}